mksusecd

```diff
@@ -2970,7 +2970,7 @@ sub add_modules_to_initrd
     mkdir "$tmp_dir/lib/modules/$kernel->{version}", 0755;
     mkdir "$tmp_dir/lib/modules/$kernel->{version}/initrd", 0755;
 
-    for ("loop.ko", "squashfs.ko") {
+    for ("loop.ko", "squashfs.ko", "lz4_decompress.ko") {
```

Comment:
It's really an edge case. It's nothing really configurable, used eactly once in the code and hasn't
changed for years. But if the list gets longer (which I don't think) , it's probably a good
idea to place it more prominently.